Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -917,20 +506,22 @@ def _exec_editor_multitest(self, request: _pytest.fixtures.FixtureRequest,
                     results[test_spec.__name__] = Result.Pass(test_spec, output, editor_log_content)
             else:
                 # Scrape the output to attempt to find out which tests failed.
-                # This function should always populate the result list, if it didn't find it, it will have "Unknown" type of result
+                # This function should always populate the result list.
+                # If it didn't find it, it will have "Unknown" type of result.
                 results = self._get_results_using_output(test_spec_list, output, editor_log_content)
-                assert len(results) == len(test_spec_list), "bug in _get_results_using_output(), the number of results don't match the tests ran"
+                assert len(results) == len(test_spec_list), (
+                    "bug in _get_results_using_output(), the number of results don't match the tests ran")
 
                 # If the editor crashed, find out in which test it happened and update the results
-                has_crashed = return_code != EditorTestSuite._TEST_FAIL_RETCODE
+                has_crashed = return_code != EditorTestSuite.test_fail_return_code
```

Comment:
as per PEP8 this is constant should be UPPER_SNAKE_CASE keep it as _TEST_FAIL_RETURN_CODE as it was before. _ to indicate that is a private variable.
https://peps.python.org/pep-0008/#constants